packages/odyssey-react-mui/src/Breadcrumbs.tsx

```diff
@@ -0,0 +1,199 @@
+  }
+
+  // breadcrumbType === "listItem" is the default
+  // Provided here without a conditional to get TS to be quiet
+  // about potential undefined returns
+  return <ButtonBase href={href}>{breadcrumbContent}</ButtonBase>;
+};
+
+const breadcrumbProviderValue: Record<
+  BreadcrumbType,
+  { breadcrumbType: BreadcrumbType }
+> = {
+  currentPage: {
+    breadcrumbType: "currentPage",
+  },
+  listItem: {
+    breadcrumbType: "listItem",
+  },
+  menuItem: {
+    breadcrumbType: "menuItem",
+  },
+};
+
+const BreadcrumbList = ({
+  children,
+  homeHref,
+  maxVisibleItems: maxVisibleItemsOverride,
+}: BreadcrumbsProps) => {
+  const maxVisibleItems = maxVisibleItemsOverride ?? children.length;
+
+  const { t } = useTranslation();
+
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+
+  const breadcrumbSections = useMemo(() => {
+    if (children.length <= maxVisibleItems) {
+      return {
+        beforeMenu: [],
+        insideMenu: [],
+        remainingBreadcrumbs: children,
+      };
+    } else {
+      const menuStart = Math.floor(maxVisibleItems / 2);
+      const menuLength = children.length - maxVisibleItems;
+
+      return {
+        beforeMenu: children.slice(0, menuStart),
+        insideMenu: children.slice(menuStart, menuStart + menuLength),
+        remainingBreadcrumbs: children.slice(menuStart + menuLength),
+      };
+    }
+  }, [children, maxVisibleItems]);
+
+  const onMenuButtonClick = useCallback<MouseEventHandler<HTMLButtonElement>>(
+    (event) => setAnchorEl(event.currentTarget),
+    []
+  );
+  const onCloseMenu = useCallback(() => {
+    setAnchorEl(null);
+  }, []);
+
+  return (
+    <MuiBreadcrumbs
+      maxItems={children.length + 1}
+      aria-label={t("breadcrumbs.label.text")}
+    >
+      {homeHref && (
+        <ButtonBase href={homeHref} aria-label={t("breadcrumbs.home.text")}>
+          <HomeIcon />
+        </ButtonBase>
+      )}
+
+      {breadcrumbSections.beforeMenu.map((breadcrumb) => (
+        <BreadcrumbContext.Provider value={breadcrumbProviderValue.listItem}>
+          {breadcrumb}
+        </BreadcrumbContext.Provider>
+      ))}
+
+      {breadcrumbSections.insideMenu && (
+        <>
+          <ButtonBase onClick={onMenuButtonClick}>…</ButtonBase>
+          <Menu
+            open={Boolean(anchorEl)}
+            onClose={onCloseMenu}
+            anchorEl={anchorEl}
+            anchorOrigin={{ horizontal: "left", vertical: "bottom" }}
+            MenuListProps={{
+              sx: {
+                minWidth: 180,
+              },
+            }}
+          >
+            <BreadcrumbContext.Provider
+              value={breadcrumbProviderValue.menuItem}
+            >
+              {breadcrumbSections.insideMenu}
+            </BreadcrumbContext.Provider>
+          </Menu>
+        </>
+      )}
+
+      {breadcrumbSections.remainingBreadcrumbs.map((breadcrumb, i) => {
+        if (i === breadcrumbSections.remainingBreadcrumbs.length - 1) {
```

Comment:
**NON-BLOCKING**

Can we use `index` instead of `i`?
```suggestion
      {breadcrumbSections.remainingBreadcrumbs.map((breadcrumb, index) => {
        if (index === breadcrumbSections.remainingBreadcrumbs.length - 1) {
```